### PR TITLE
Add manifest file for bugzilla backporter

### PR DESCRIPTION
**Pull request by GeoBK**

### CODE REVIEW

File: clusters/app.ci/assets/bugzilla-backporter.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+              path: /healthz/ready
+              port: 8081
+            initialDelaySeconds: 10
+            periodSeconds: 3
+            timeoutSeconds: 600
+          volumeMounts:
+          - name: bugzilla
+            mountPath: /etc/bugzilla
+            readOnly: true
+          resources:
```
Comment (GeoBK) [inline clusters/app.ci/assets/bugzilla-backporter.yaml:104]: I dont think I'll need this much resources. What would be the right amount?

File: clusters/app.ci/assets/bugzilla-backporter.yaml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            requests:
+              memory: "2Gi"
+              cpu: "200m"
+        volumes:
+        - name: bugzilla
+          secret:
+            secretName: bugzilla-credentials-openshift-bugzilla-robot
```
Comment (GeoBK) [inline clusters/app.ci/assets/bugzilla-backporter.yaml:111]: What is this robot? Will I need to make any more configurations for this?

Comment (alvaroaleman) [reply clusters/app.ci/assets/bugzilla-backporter.yaml:111]: No, the secret is already there

File: clusters/app.ci/assets/bugzilla-backporter.yaml (hunk 1 above)
Comment (alvaroaleman) [reply clusters/app.ci/assets/bugzilla-backporter.yaml:104]: Thats always a tricky question :D Usually we want out by running it for some time and then looking at prometheus data on how much it used

Comment (GeoBK) [reply clusters/app.ci/assets/bugzilla-backporter.yaml:104]: Ok. So 2GB(I am guessing thats what 2Gi means) and 200m is good ?

File: clusters/app.ci/assets/bugzilla-backporter.yaml [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  metadata:
+    namespace: ci
+    labels:
+      app: prow
+      component: bugzilla-backporter
+    name: bugzilla-backporter
+  spec:
+    ports:
+    - name: main
+      port: 80
```
Comment (GeoBK) [inline clusters/app.ci/assets/bugzilla-backporter.yaml:30]: The port for the deployed container is 8080. Does this need to match that?

Comment (alvaroaleman) [reply clusters/app.ci/assets/bugzilla-backporter.yaml:30]: no, the `targetPort` needs to match that and it does, the port named `main` is 8080

**Pull request merged**